### ISSUE TITLE
bug: on-scroll does not call digest cycle

### ISSUE THREAD

**Issue by quanghoc** [driftyco/ionic#3848]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>all</span>  

<span ionic-description>**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>all</span>  

<span ionic-description>I posted a question here

http://stackoverflow.com/questions/30493021/ng-style-wont-bind-with-ionic-scroll-position

Here is the solution

http://codepen.io/pramodsankarl/pen/oXYOBb

Basically

```
  $scope.isScrolling = function() {
    console.log($ionicScrollDelegate.getScrollPosition());
    $scope.pos.x = ($ionicScrollDelegate.getScrollPosition().top / 2204 * 100) + '%';
    
    $scope.$evalAsync();
  }
```
</span>

<span is-issue-template></span>
</span>

<span is-issue-template></span>

**Comment by Ionitron** [comment]
Greetings @quanghoc!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=3848). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=3848)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by ptarjan** [comment]
+1

**Comment by Ionitron** [comment]
Greetings @quanghoc!

I've closed this issue because my sensors indicated it was old and inactive, and may have already been fixed in recent versions of Ionic. However, if you are still experiencing this issue, please feel free to reopen this issue by creating a new one, and include any examples and other necessary information, so that we can look into it further.

Thank you for allowing me to assist you.
